Translation Editor/make_translation.py

```diff
@@ -7,6 +7,13 @@
 from datetime import datetime
 import sys
 import fontTables
+
+# To cause no confusion with the original firmware version of TS100,
+# which actually has the latest firmware version 2.18
+# Lets go ahead start with version 2.20.0001 BETA
+# Version String: xx.yy.zzzz TAG --> x:Major y:Minor z:Build TAG: REL | BETA | ALPHA
+BUILD_VERSION = 'V2.20.0001 BETA'

```

Comment:
Any chance you feel like swapping out the REL/Beta/Alpha + build numbers to instead be major.minor.`git short hash`

To make it clear where it came from ?

Reply:
@GeminiServer

Reply:
Good idea to use the git short hash instead a build number.